docs/life-of-a-request.md

```diff
@@ -60,15 +70,16 @@ gets translated in Chinese at `zh-cn/hello/` (preview [here](https://zulip.com/z
 
 Note the `zh-cn` prefix--that url pattern gets added by `i18n_patterns`.
 
-## API endpoints use REST
+## API endpoints use [REST](http://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm)
+
+Our example is a REST API endpoint. It's a PUT to `/users`.
 
-With the exception of Webhooks (which we do not control), and legacy
+With the exception of Webhooks (which we do not control), legacy
 endpoints, and logged-out endpoints, Zulip uses REST for its API. This
 means that we use:
 
-* POST for creating something new where you don't have a unique ID
-* PUT for creating or completely replacing something for which you have
-  a unique ID
+* POST for creating something new where we don't have a unique ID
```

Comment:
It's also at the end of the list, but I'll move it up so it's easier to discover.